src/components/organisms/SignForm.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react';
+import {Text, ScrollView, ActivityIndicator} from 'react-native';
+import {ToSectionMessage} from '../atoms';
+import {LogInFields, SignUpFields, AlternatedSignIn} from '../molecules';
+import {Overlay} from 'react-native-elements';
+import {styles, CustomOverlayStyle} from '../../styles';
+import Theme from '../../theme/light';
+
+const SignFields = ({type}) => {
+  return type === 'signin' ? <LogInFields /> : <SignUpFields />;
+};
+
+const SignForm = ({type, toSectionMessage}) => {
+  const [isVisible, setIsVisible] = useState(false);
+  return (
+    <ScrollView style={styles.signScreenContainer}>
+      <SignFields type={type} />
+      <AlternatedSignIn
+        isVisible={isVisible}
+        setIsVisible={setIsVisible}
+        title={type === 'signup' ? 'Sign up with Google' : 'Log in with Google'}
+      />
+      <ToSectionMessage
+        text={toSectionMessage.text}
+        link={toSectionMessage.link}
+      />
+      <Overlay
+        overlayStyle={CustomOverlayStyle.ModalContainer}
+        isVisible={isVisible}
+        onBackdropPress={() => setIsVisible(!isVisible)}>
+        <ActivityIndicator size={64} color="#5566dd" />
```

Comment:
You can use the Theme.Colors library that you have imported to replace it with the color code, so that the hexadecimal code is not seen. Placing the color in your library.

Reply:
we will make a issue to fix colors references's errors tomorrow